### PR TITLE
40/style deposit widget

### PR DESCRIPTION
**Pull request by anxolin**
This PR adds some styles, specially for the deposit page, but also styles the pages layout.

Pages layout:
image

Withdraw layout (widget):
image

**Test**
https://pr50--dexreact.review.gnosisdev.com

**Fixes**: #40

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr50--dexreact.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr50--dexreact.review.gnosisdev.com

File: src/components/layout/Header.tsx [hunk 1]
```diff
@@ -33,8 +33,8 @@ const Wrapper = styled.header`
   .connect-wallet {
     padding: 1rem;
     margin: 1rem;
-    border: 2px solid var(--color-btn-border);
-    color: var(--color-btn-border);
+    border: 2px solid #ff5097;
+    color: #ff5097;
```
Comment (alfetopito) [inline src/components/layout/Header.tsx]: Why not use the colours from the global styles?

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr50--dexreact.review.gnosisdev.com

File: src/components/layout/Header.tsx (hunk 1 above)
Comment (anxolin) [reply src/components/layout/Header.tsx]: I got tired of modifying in two places just for modifying a color of a scoped component
I think vars make sense if we think we will use them a lot.
I just left 

```
--color-text-primary: #3d414c;
--color-text-secondary: #8c8c8c;
```

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr50--dexreact.review.gnosisdev.com

**Pull request merged**